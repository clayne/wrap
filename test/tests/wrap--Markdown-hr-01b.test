wrap | /dev/null | -u | md-hr-01b.md | 0
