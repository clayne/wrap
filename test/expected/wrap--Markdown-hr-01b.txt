---
That was a horizontal rule.
